src/main/java/com/twelfthmile/yuga/Yuga.java

```diff
@@ -1295,6 +1297,25 @@ private static void setIfNumRange(String str, int i, FsaContextMap map) {
         }
     }
 
+    private static String checkIfData(String str, int j, FsaContextMap map) {
+        String sData = "";
+        switch (str.charAt(j)){
+            case 'k':
+                map.setVal("data_type","KB");
+                sData = " KB";
```

Comment:
You can return inside the `case` block and avoid `break` and `sData`

Reply:
This method's return type was string earlier but now its void and I am not returning anything.